src/rules/no-stale-state-across-await.ts

```
@@ -0,0 +1,182 @@
+            setterCallsByName.set(call.name, []);
+          }
+          setterCallsByName.get(call.name)!.push(call);
+        }
+
+        // Check each setter for violations
+        for (const [setterName, calls] of setterCallsByName) {
+          if (calls.length < 2) continue; // Need at least 2 calls to have a violation
+
+          // Check if there are calls both before and after any async boundary
+          for (const boundary of asyncBoundaries) {
+            const callsBefore = calls.filter(call => call.position < boundary.position);
```

ESLint

[error] Replace `call·=>·call.position·<·boundary.position` with `⏎··············(call)·=>·call.position·<·boundary.position,⏎············`

(prettier/prettier)